projects/components/src/common/interfaces/action-item.interface.ts

```diff
@@ -177,3 +144,58 @@ export enum TextIcon {
     TEXT = 1 << 1,
     ICON_AND_TEXT = TextIcon.ICON | TextIcon.TEXT,
 }
+
+/**
+ * This is created separately from {@link ContextualActionInlineDisplayConfig} because, featured count is only required
+ * when contextual actions are displayed as a dropdown
+ */
+export interface ContextualActionDropdownDisplayConfig {
+    /**
+     * How many buttons should display on the featured section.
+     *
+     * Used when you want to set a limit on the number of featured buttons shown.
+     *
+     * If featuredCount is not set, it will default to all featured actions.
+     */
+    featuredCount: number;
+    /**
+     * How the featured actions should be displayed
+     */
+    styling: ActionStyling.DROPDOWN;
+    /**
+     * If the title should be the button label, icon, or both
+     * Defaults to ICON if unset.
+     */
+    buttonContents: TextIcon;
+}
+
+/**
+ * This along with {@link ContextualActionDropdownDisplayConfig} is one of the types of
+ * {@link ActionDisplayConfig.contextual}
+ */
+export interface ContextualActionInlineDisplayConfig {
```

Comment:
Can you add a comment explaining what INLINE and DROPDOWN mean? I was confused about this.